styles/styles.css

```
@@ -97,36 +104,230 @@ body {
   font-weight: 700;
 }
 
-.services-list {
+/*Блог сервиса*/
+.services-item {
   text-align: center;
-  background-color: #83b3d31f;
+  background-color: rgba(131, 179, 211, 0.12);
   color: #000000;
-  text-decoration: none;
+}
+.background-white {
+  background-color: #ffffff;
+}
+
+.background-blue {
+  background-color: rgba(131, 179, 211, 0.2);
 }
 .services-title {
   font-size: 24px;
   line-height: 28px;
   font-weight: 700;
   text-transform: uppercase;
 }
-/*не работает фон*/
-.info-email{
-    color:#ffffff;
-    background-color: black;
-    background-image: url("/images/background-footer.jpg");
-    text-align: center;
+
+.info-hotel {
+  text-align: center;
+  background-color: #ffffff;
+}
+/*Блог подписки*/
+.info-email {
+  color: #ffffff;
+  background-color: #000000;
+  background-image: url("/images/background-footer.jpg");
+  background-size: 100% auto;
+  background-repeat: no-repeat;
+  text-align: center;
+}
+.info-email-title {
```

```suggestion
}

.info-email-title {
```
По стайл гайду, между правилами должны быть пробелы